src/common/k8s-api/kube-api.ts

```diff
@@ -479,7 +478,7 @@ export class KubeApi<T extends KubeObject> {
         });
       })
       .catch(error => {
-        if (error instanceof DOMException) return; // AbortController rejects, we can ignore it
+        if (typeof DOMException === "function" && error instanceof DOMException) return; // AbortController rejects, we can ignore it
```

Comment:
@Nokel81 do you remember why we need to handle `DOMException`?

Reply:
DOMException is the type of exception that the `abort-controller` package throws when the `cancel` function is called.